pylibjpeg/tests/test_handler.py

```diff
@@ -265,15 +265,21 @@ def test_3s_1f_ybr_422(self):
 
         #self.plot(arr)
 
-        # Reference values from GDCM handler - slightly different
+        # Reference values may be slightly different depending on handler
+        # Pillow: assert ( 76,  86, 251) == tuple(arr[ 5, 50, :])
         assert ( 76,  85, 254) == tuple(arr[ 5, 50, :])
-        # assert (166, 109, 190) == tuple(arr[15, 50, :])
-        assert (166, 108, 190) == tuple(arr[15, 50, :])  # Different
+        # GDCM:   assert (166, 109, 190) == tuple(arr[15, 50, :])
+        # Pillow: assert (166, 112, 185) == tuple(arr[15, 50, :])
+        assert (166, 108, 190) == tuple(arr[15, 50, :])
```

Comment:
I don't know what's going on with Pillow. In theory it should be identical to GDCM...